src/logic/operate.js

```diff
@@ -0,0 +1,14 @@
+import Big from 'big.js';
+
+// eslint-disable-next-line consistent-return
+const operate = (numberOne, numberTwo, operation) => {
+  const operand1 = Big(numberOne);
+  const operand2 = Big(numberTwo);
+
+  if (operation === '-') return operand1.minus(operand2);
+  if (operation === '+') return operand1.plus(operand2);
+  if (operation === 'X') return operand1.times(operand2);
+  if (operation === '÷') return operand1.div(operand2);
```

Comment:
**Optional**
This is a nice situation to use a switch/case statement. This will also allow you to remove the disabled ESlint rule on line 3.